docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>I'm confused: is the model conditionned on data here or not?</li><li>Can you explain why you have to define the MvGaussian like that: why not generating data from <code>np.random.multivariate_normal</code> ?</li><li>Related to my question above: why is the use of <code>Potential</code> needed here?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
No data here, we are just defining a potential and sampling from it. There is "no need" to do this, but doing this is handy and it is also related to an example in one of the SMC papers.